go.mod

```diff
@@ -89,7 +88,9 @@ require (
 	github.com/hashicorp/go-uuid v1.0.2 // indirect
 	github.com/ianlancetaylor/cgosymbolizer v0.0.0-20170921033129-f5072df9c550 // indirect
 	github.com/jackc/fake v0.0.0-20150926172116-812a484cc733 // indirect
+	github.com/jackc/pgproto3/v2 v2.0.2
 	github.com/jackc/pgx v3.6.2+incompatible
+	github.com/jackc/pgx/v4 v4.7.1
```

Comment:
this won't be an issue -- older pgx imports will import the old library (as go.mod intended).